### PR TITLE
Security page clean up

### PR DESCRIPTION
**Pull request by Nicca42**
Cleaning up + adding more info

### CODE REVIEW

File: src/docs/security-model/bounties.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Optimism has a very detailed [Bug bounty Page on Immunefi](https://immunefi.com/bounty/optimism/). In the listing you can find all the information relating to assets in scope, reporting, and the payout process. 
+
+### Bedrock Bounty
+
+With our upcoming launch of [Bedrock](../developers/bedrock.md) we will be launching a specific buy bounty program. Follow our [Twitter](https://twitter.com/optimismFND) or keep an eye on [Immunefi](https://immunefi.com/bounty/optimism/) for more information.
+
+### Unscoped Bug
+
+If you think you have found a critical or major bug that is not covered by our existing bug bounty, please email us at **security@optimism.io** and we will get back to you.
```
Comment (maurelian) [inline src/docs/security-model/bounties.md:20]:
```suggestion
If you think you have found a critical or major bug that is not covered by our existing bug bounty, please report it to us via the Immunefi program regardless.
```

File: src/docs/security-model/README.md [hunk 2]
```diff
@@ -3,35 +3,12 @@ title: Optimism's Security Model
 lang: en-US
 ---
 
-The Optimism blockchain is a work in progress.
-Constantly pushing to improve the security guarantees that users have while using Optimism is a top priority.
-At the moment, **it’s important to understand that the security of the Optimism blockchain is dependent on a [multisig wallet](https://www.coindesk.com/tech/2020/11/10/multisignature-wallets-can-keep-your-coins-safer-if-you-use-them-right/)** managed by several anonymous individuals.
-This multisig wallet can be used to upgrade core Optimism smart contracts without upgrade delays.
+Security in crypto is vital. Users, developers and infrastructure providers all need to trust that the code works, as expected. When it breaks, these same groups (rightfully) demand a fast and secure fix that is clearly communicated.
```
Comment (maurelian) [inline src/docs/security-model/README.md:6]: Some nitty suggestions:

```suggestion
Security in crypto is vital. Users, developers and infrastructure providers expect the code to work as intended. When it doesn't, these same groups (rightfully) demand a fast, secure, and clearly communicated fix. 
```

Rationale: 
1. I wanted to remove the word 'trust', as I find it quite confusing in this context. Y'know, like does it meet you "trust the code", or "trust us to write secure code". Easier to just work around it. 
2. Expected/intended are similar... but my thought is that most users expectations are vaguely defined. I feel like saying 'intended' is more suggestive of the developers intent.
3. Reordered the last sentence just a bit for flow.

File: src/docs/security-model/README.md [hunk 3]
```diff
@@ -3,35 +3,12 @@ title: Optimism's Security Model
...
+Security in crypto is vital. Users, developers and infrastructure providers all need to trust that the code works, as expected. When it breaks, these same groups (rightfully) demand a fast and secure fix that is clearly communicated. 
 
-Please also keep in mind that just like any other system, **the Optimism codebase may contain unknown bugs** that could lead to the loss of some or all of the assets held within the system.
-[Optimism’s smart contract codebase has been audited repeatedly](https://github.com/ethereum-optimism/optimism/tree/develop/technical-documents/audits) but **audits are not a stamp of approval** and **a completed audit does not mean that the audited codebase is free of bugs.**
-It’s important to understand that using Optimism inherently exposes you to the risk of bugs within the Optimism codebase.
+Over the years at Optimism we have learnt a great deal about this delicate balance, and the importance of transparency around security models. It is not good enough to just be transparent, it also needs to be accessible. If the average user cannot understand your security model then it does not matter how transparent it is.
```
Comment (maurelian) [inline src/docs/security-model/README.md:8]: Nit: 
~~learnt~~ learned
I think we prefer American english

File: src/docs/security-model/README.md [hunk 4]
```diff
@@ -3,35 +3,12 @@ title: Optimism's Security Model
...
 
-Please also keep in mind that just like any other system, **the Optimism codebase may contain unknown bugs** that could lead to the loss of some or all of the assets held within the system.
-[Optimism’s smart contract codebase has been audited repeatedly](https://github.com/ethereum-optimism/optimism/tree/develop/technical-documents/audits) but **audits are not a stamp of approval** and **a completed audit does not mean that the audited codebase is free of bugs.**
-It’s important to understand that using Optimism inherently exposes you to the risk of bugs within the Optimism codebase.
+Over the years at Optimism we have learnt a great deal about this delicate balance, and the importance of transparency around security models. It is not good enough to just be transparent, it also needs to be accessible. If the average user cannot understand your security model then it does not matter how transparent it is. 
 
-## Security Model FAQ
+In line with this philosophy of radical transparency we have a very accessible run down of [Optimism's security model](./optimism-security-model.md) that anyone can understand.
```
Comment (maurelian) [inline src/docs/security-model/README.md:10]:
```suggestion
In line with this philosophy of transparency we have a very accessible run down of [Optimism's security model](./optimism-security-model.md) that anyone can understand. 
```

The term "radical" wasn't used in the previous paragraph, and I think it has a very strong (but not clear to me) connotation about transparency that I'd prefer we not necessarily be held to.

File: src/docs/security-model/bounties.md (hunk 1 above)
Comment (Nicca42) [reply src/docs/security-model/bounties.md:20]: No seems good, adding.

File: src/docs/security-model/README.md (hunk 4 above)
Comment (Nicca42) [reply src/docs/security-model/README.md:10]: I could link to the radical transparency ideology, but I don't think we follow it too closely so we can 100% remove it.

**Comment by Nicca42** [comment]
@maurelian anything else? Otherwise, I think this is a strong start to the security page. Feel free to add to this as our bounties/processes grow.

**Review by maurelian — APPROVED** [review]
LGTM

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**